src/UI/Internal/Text.elm

```diff
@@ -365,3 +367,28 @@ combinedAttrs { ellipsis } =
 
     else
         []
+
+
+ellipsizedText : RenderConfig -> TextSize -> String -> Element msg
+ellipsizedText cfg size content =
+    let
+        lineHeightSize =
+            lineHeight (isMobile cfg) size
+    in
+    content
+        |> Html.text
+        |> Element.html
+        |> Element.el
+            (ellipsisAttrs lineHeightSize)
+
+
+ellipsisAttrs : Int -> List (Attribute msg)
```

Comment:
I don't think this should be external, as we should not use text without use `UI.Text`, if you think it makes sense be internally exposed, I can move to Internal.Utils.Element

Reply:
There are a lot of texts in internal components that don't fit any of our styles, like Filter's Header, Buttons, etc;